hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -1507,6 +1507,91 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
```

Comment:
Every entry is repeated twice in this file. Need to do the same for this entry.

Reply:
Done. I've added it. Great catch.